From: Ismael Luceno <ismael@iodev.co.uk>

Fix definition of off_t type to match the .c files; it does not make sense to
use long long.

--- a/vbisam.h
+++ b/vbisam.h
@@ -22,13 +22,7 @@
 
 #include <sys/types.h>
 
-#define	VBISAM_USE_LONG_LONG	/* Note hack for now */
-
-#ifdef	VBISAM_USE_LONG_LONG
-#define	vbisam_off_t	long long
-#else
-#define	vbisam_off_t	int
-#endif
+#define	vbisam_off_t	off_t
 
 #include	<vbdecimal.h>
 
